1-foundations/4-api-and-dataviz/foundations-s4-api.ipynb

```diff
@@ -373,6 +373,17 @@
     "    return None, None"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run this line to verify that your solution works. It should not return an error\n",
```

Comment:
Consider add `%%script echo Remove this line after filling in your own code` at the top of this cell so the whole notebook can be executed without error.

Reply:
added in 09b9665